broker-daemon/block-order-worker/index.js

```diff
@@ -100,6 +100,41 @@ class BlockOrderWorker extends EventEmitter {
     }
 
     const blockOrder = new BlockOrder({ id, marketName, side, amount, price, timeInForce })
+    const baseEngine = this.engines.get(blockOrder.baseSymbol)
+    const counterEngine = this.engines.get(blockOrder.counterSymbol)
+    const { address: counterSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.counterSymbol})
+    const { address: baseSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.baseSymbol})
```

Comment:
I would run this in parallel to the other call to the payment channel network service